service/start.go

```diff
@@ -84,6 +84,7 @@ func (dependency *Dependency) Start(service *Service, details dependencyDetails,
 		Image: dependency.Image,
 		Args:  strings.Fields(dependency.Command),
 		Env: []string{
+			"MESG_SERVICE_HASH=" + service.Hash(),
```

Comment:
I think we should rename it `MESG_SERVICE_ID`

Reply:
We could also rename the function `hash` to `id`.
Even if technically it's a hash of the service, we are using it as a ID. So it easier to understand if it named `id`